docs/introduction-to-smart-contracts.rst

```diff
@@ -260,6 +260,10 @@ blocks that are added on top, the less likely it is. So it might be that your tr
 are reverted and even removed from the blockchain, but the longer you wait, the less
 likely it will be.
 
+.. note::
+    Transactions are not guaranteed to happen on the next block or any future specific block, since it is up to the miners to include transactions and not the submitter of the transaction. This applies to function calls/transactions and contract creation transactions.
```

Comment:
Perhaps `This applies both to function call and contract creation transactions.`